src/SFA.DAS.AdminService.Web/Validators/Roatp/Applications/RoatpFinancialClarificationViewModelValidator.cs

```diff
@@ -1,93 +1,158 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
 using FluentValidation;
 using FluentValidation.Validators;
+using Microsoft.AspNetCore.Http;
+using SFA.DAS.AdminService.Common.Validation;
 using SFA.DAS.AdminService.Web.ViewModels.Apply.Financial;
 using SFA.DAS.AdminService.Web.ViewModels.Roatp.Financial;
 using SFA.DAS.AssessorService.ApplyTypes.Roatp.Apply;
 
 namespace SFA.DAS.AdminService.Web.Validators.Roatp.Applications
 {
-    public class RoatpFinancialClarificationViewModelValidator : AbstractValidator<RoatpFinancialClarificationViewModel>
+    public class RoatpFinancialClarificationViewModelValidator : IRoatpFinancialClarificationViewModelValidator
     {
-        public RoatpFinancialClarificationViewModelValidator()
+        private const long MaxFileSizeInBytes = 5 * 1024 * 1024;
+        private const string MaxFileSizeExceeded = "The selected file must be smaller than 5MB";
+        private const string FileMustBePdf = "The selected file must be a PDF";
+        private const string ClarificationFile = "ClarificationFile";
+        private const string ClarificationResponse = "ClarificationResponse";
+
+        public async Task<ValidationResponse> Validate(RoatpFinancialClarificationViewModel vm,
+            bool isClarificationFileUpload, bool isClarificationOutcome)
```

Comment:
The interface has this as `isClarificationFilesUpdate` rather than `isClarificationFileUpload` could they both be consistent?